swabbie-aws/src/main/kotlin/com/netflix/spinnaker/swabbie/aws/images/AmazonImageHandler.kt

```diff
@@ -91,7 +91,7 @@ class AmazonImageHandler(
               "imageIds" to markedResources.map { it.resourceId }.toSet(),
               "cloudProvider" to AWS,
               "region" to workConfiguration.location,
-              "stageTimeoutMs" to Duration.ofMinutes(15).toMillis().toString()
+              "stageTimeoutMs" to Duration.ofMinutes(40).toMillis().toString()
```

Comment:
This is fine and correct, in that Orca's `MonitorDeleteImageTask` is a `OverridableTimeoutRetryableTask` which will honor the longer stageTimeoutMs parameter. However, the swabbie -> orca -> clouddriver trifecta may pose major operational challenges in an environment where there are > 100k images to delete, orca and clouddriver are resource constrained, AWS describeImage calls are extremely slow, and swabbie activity is limited to what happens to be peak spinnaker usage hours.

* `MonitorDeleteImageTask` has a backoff period of 5 seconds. If these may actually run for ~30 minutes, we can calculate how many can be queued concurrently based on current capacity of orca and our clouddriver shard for orchestrations.
* Consider making image deletion fire-and-forget from the perspective of Swabbie. This could preserve the existing orca stage (minus the monitor task) and clouddriver deleteImage kato task. Swabbie could optionally mark a resource as "deleting" and reconcile as deleted or re-send a delete request on a future crawl of all resources. I prefer this approach over the current PR.
* If there are major blockers to a fire-and-forget approach, we may need to consider running a dedicated orca cluster/queue for swabbie, and figure out clouddriver resourcing for it.